system_metrics_collector/src/system_metrics_collector/linux_cpu_measurement_node.hpp

```diff
@@ -69,6 +69,13 @@ class LinuxCpuMeasurementNode : public system_metrics_collector::PeriodicMeasure
   virtual ~LinuxCpuMeasurementNode() = default;
 
 protected:
+  /**
+   * Creates a ROS2 timer with a period of measurement_period_.
+   *
+   * @return if setup was successful
+   */
+  bool setupStart() override;
```

Comment:
If last_measurement_ is initialized with a default value or in the constructor then we don't need to override.

Reply:
Continue this conversation in https://github.com/ros-tooling/system_metrics_collector/pull/16#discussion_r355054133.